scripts/utils/board_v1_to_v2.py

```diff
@@ -75,7 +75,8 @@ def board_v1_to_v2(board_root, board, new_board, group, vendor, soc, variants):
         }
     else:
         with open(board_settings_file) as f:
-            board_settings = ruamel.yaml.safe_load(f) # pylint: disable=assignment-from-no-return
+            yaml = ruamel.yaml.YAML(typ='safe', pure=True)
+            board_settings = yaml.load(f) # pylint: disable=assignment-from-no-return
```

Comment:
can you explain why this is needed?

Reply:
$ python3 ./scripts/utils/board_v1_to_v2.py -b frdm_k64f -g blob -v blob -s k6x

New board already exists, updating board with additional SoC
Moving files to the new board folder...
Creating or updating board.yaml...
Traceback (most recent call last):
  File "/home/yves/sw/zephyr/zephyrproject/zephyr/./scripts/utils/board_v1_to_v2.py", line 206, in <module>
    board_v1_to_v2(
  File "/home/yves/sw/zephyr/zephyrproject/zephyr/./scripts/utils/board_v1_to_v2.py", line 78, in board_v1_to_v2
    board_settings = ruamel.yaml.safe_load(f) # pylint: disable=assignment-from-no-return
  File "/home/yves/.local/lib/python3.10/site-packages/ruamel/yaml/main.py", line 1105, in safe_load
    error_deprecation('safe_load', 'load', arg="typ='safe', pure=True")
  File "/home/yves/.local/lib/python3.10/site-packages/ruamel/yaml/main.py", line 1039, in error_deprecation
    raise AttributeError(s, name=None)
AttributeError: 

"safe_load()" has been removed, use
  yaml = YAML(typ='safe', pure=True)

  yaml.load(...)



instead of file "/home/yves/sw/zephyr/zephyrproject/zephyr/./scripts/utils/board_v1_to_v2.py", line 78



            board_settings = ruamel.yaml.safe_load(f) # pylint: disable=assignment-from-no-return